storage/rocksdb/rdb_datadic.cc

```diff
@@ -4097,6 +4097,12 @@ bool Rdb_dict_manager::init(rocksdb::DB *const rdb_dict,
   m_db = rdb_dict;
 
   m_system_cfh = cf_manager->get_or_create_cf(m_db, DEFAULT_SYSTEM_CF_NAME);
+  rocksdb::ColumnFamilyHandle *default_cfh = cf_manager->get_or_create_cf(m_db, DEFAULT_CF_NAME);
```

Comment:
let's call `get_cf` here instead of `get_or_create_cf`, it makes it clearer that the default column family should already be created by this point.  Maybe add a comment too.